src/fuse.js

```diff
@@ -55,7 +65,8 @@ define([
 
             return {
                 id: mug.ufid,
-                name: mug.absolutePath,
+                name: mug.absolutePath || mug.hashtagPath,
```

Comment:
non-blocking: when would a mug have empty `absolutePath` and non-empty `hashtagPath`?